fixing extended trial dates (#9316)
fishtown-analytics/docs.getdbt.com: website/snippets/_dbt-state-vs-sao.md

- **Smarter data freshness tracking** — dbt State tracks data freshness across the <Term id="dag">DAG</Term> and automatically propagates it through models materialized as views. Unlike state-aware orchestration's `build_after` config which compares against the model's last successful execution, dbt State's `lag_tolerance` compares against the freshness of the underlying data.
- **Advanced change detection** — dbt State can detect and ignore file modifications that don't change actual transformation logic, such as adding a comment or cleaning up whitespace.

> If you were using state-aware orchestration prior to June 1, 2026, you can continue using it. Your dbt State trial will be extended at no cost until the billing period begins on September 1, 2026. If your trial wasn't extended, contact your account team. For details on billing after the trial ends, refer to [dbt State usage and pricing](/docs/platform/billing#dbt-state-usage). 

Reviewer: ```suggestion
If you were using state-aware orchestration prior to June 1, 2026, you can continue using it. Your dbt State trial will be extended until the billing period begins on September 1, 2026. If your trial wasn't extended, contact your account team. For details on billing after the trial ends, refer to [dbt State usage and pricing](/docs/platform/billing#dbt-state-usage). 
```

> If you were using state-aware orchestration prior to June 1, 2026, you can continue using it. As an existing customer, your dbt State trial will be extended until the billing period begins on September 1, 2026. If you don't see the extension, contact your account team. For details on billing after the trial ends, refer to [dbt State usage and pricing](/docs/platform/billing#dbt-state-usage). 

Reviewer: ```suggestion
If you were using state-aware orchestration prior to June 1, 2026, you can continue using it. Your dbt State trial will be extended at no cost until the billing period begins on September 1, 2026. If your trial wasn't extended, contact your account team. For details on billing after the trial ends, refer to [dbt State usage and pricing](/docs/platform/billing#dbt-state-usage). 
```
